### PR TITLE
feat: product demo page

### PR DESCRIPTION
**Pull request by pro-dev-ryan**
product details page accomplished

### CODE REVIEW

File: components/Demo/ProductScreenshots.vue [hunk 1]
```diff
@@ -41,35 +41,35 @@
 </template>
 
 <script>
-  // import { Swiper, SwiperSlide } from "vue-awesome-swiper";
-  // import "swiper/css/swiper.css";
+import { Swiper, SwiperSlide } from "swiper/vue";
```
Comment (devzakir) [inline components/Demo/ProductScreenshots.vue:44]: remove the import

File: components/Demo/TestimonialSection.vue [hunk 2]
```diff
@@ -50,8 +50,8 @@
 </template>
 <script>
 
-// import { Swiper, SwiperSlide } from "vue-awesome-swiper";
-// import "swiper/css/swiper.css";
+import { Swiper, SwiperSlide } from "swiper/vue";
```
Comment (devzakir) [inline components/Demo/TestimonialSection.vue:53]: remove the import

File: pages/demo/[slug].vue [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+  <!-- {{ product }} -->
   <div>
     <demo-header :product="product" v-if="product" />
     <product-hero :product="product" v-if="product" />
-    <why-choose-our-product :product="product" v-if="product" />
-    <div v-for="(section, index) in product.contents" :key="index">
+    <why-choose-our-product :product="product" v-if="product?.contents" />
+    <div v-for="(section, index) in product?.contents" :key="index">
       <div v-if="section.__typename == 'ExclusivefeatureRecord'">
         <exclusive-feature :data="section" />
```
Comment (devzakir) [inline pages/demo/[slug].vue:9]: remove the imports from this file.

**Review by devzakir — CHANGES_REQUESTED** [review]

**Review by devzakir — CHANGES_REQUESTED** [review]
Please fix the conflicts. and remove yarn.lock from git history

**Review by devzakir — CHANGES_REQUESTED** [review]
- [x] Design is broken, color is mismatched across the entire demo page. please check. I have added a few examples below. 

image
image

image
image

File: components/Demo/TestimonialSection.vue (hunk 2 above)
Comment (pro-dev-ryan) [reply components/Demo/TestimonialSection.vue:53]: Import removed

File: pages/demo/[slug].vue (hunk 3 above)
Comment (pro-dev-ryan) [reply pages/demo/[slug].vue:9]: Done

**Review by pro-dev-ryan — COMMENTED** [review]
All redundant imports has been removed. Design fixing in progress

**Review by pro-dev-ryan — COMMENTED** [review]
All changes has been done. Please check

**Pull request closed without merging**